### ISSUE TITLE
About learning rate schedule

### ISSUE THREAD

**Issue by theneotopia** [sthalles/SimCLR#43]
Hi, Thalles.
https://github.com/sthalles/SimCLR/blob/1848fc934ad844ae630e6c452300433fe99acfd9/run.py#L79
Shouldn't `T_max=args.epochs` instead of `T_max=len(train_loader)` since learning rate schedule happens at every epoch?
Thanks.
